src/pre_commit_vauxoo/cfg/.flake8-optional

```diff
@@ -4,17 +4,18 @@ jobs = 0
 # B = bugbear
 # B9 = bugbear opinionated (incl line length +10% tolerance)
 select = C,E,F,W,B,B9
-ignore =
-    E203  # whitespace before ':' (black behaviour)
```

Comment:
I prefer this way

Is not supported anymore?

Reply:
No, inline comments are no longer supported. 

> Following the recommended settings for [Python’s configparser](https://docs.python.org/3/library/configparser.html#customizing-parser-behaviour), Flake8 does not support inline comments for any of the keys.

https://flake8.pycqa.org/en/latest/user/configuration.html